src/Aws/DynamoDb/Streams/Core.hs

```diff
@@ -139,15 +139,19 @@ instance AwsType StreamsAction where
   toText = streamsActionToText
   parse = parseStreamsAction
 
--- | Currently, only 'UsEast1' and 'EuWest1' are supported.
---
 streamsServiceEndpoint
   ∷ Region
   → B8.ByteString
 streamsServiceEndpoint = \case
-  UsEast1 → "streams.preview-dynamodb.us-east-1.amazonaws.com"
-  EuWest1 → "streams.preview-dynamodb.eu-west-1.amazonaws.com"
-  region → error $ "Unsupported region: " ⊕ show region
+  ApNortheast1       → "streams.dynamodb.ap-northeast-1.amazonaws.com"
```

Comment:
Would you mind removing the vertical alignment from these lines? This is a style that @larskuhtz and I lovingly term "ascii art", and whilst it can be very pretty, we have preferred to keep it out of our packages.